### DISCUSSION TITLE
Bulleted list not working

### DISCUSSION THREAD

**Discussion by bkmalan** [dolanmiu/docx#961] [Q&A]
Hey, 

I was working with version 5.5.0 earlier and the bullet points were working as expected, I recently updated the package to the latest version and my bullet points are broken.

Please help me resolve this in whichever way I can.

Thanks,

**Comment by dolanmiu** [comment]
Could you show me your code so I could debug?

Check this out, this is the new v6 way:

https://github.com/dolanmiu/docx/blob/master/demo/3-numbering-and-bullet-points.ts

**Reply by bkmalan** [reply]
That link helped me resolve the issue.

Thanks for you help!

I also need help in numbering lists.

This is how I'm creating config: 

config.style.paragraph.indent.left = (360 * level);
config.style.paragraph.indent.hanging = 360;

The level is an integer to specify how deep the list is nested, it starts from 1 and increases by 1 for each nested list.

When I try this, the output I get is like in the image below.
![Screenshot from 2021-05-25 19-29-47](https://user-images.githubusercontent.com/55806149/119512371-de73bf80-bd90-11eb-89a8-452c3db74d1b.png)


The problem is the space between the numbering and the text, i want the spacing to be similar to the level 4 for all levels.

Kindly help me achieving this.

Thanks,
